WifiManager/doc/WifiPlugin.md

```diff
@@ -85,19 +85,19 @@ WifiManager interface methods:
 
 | Method | Description |
 | :-------- | :-------- |
-| [cancelWPSPairing](#method.cancelWPSPairing) | Cancels WPS pairing |
+| [cancelWPSPairing](#method.cancelWPSPairing) | Cancels the in-progress WPS pairing operation |
 | [clearSSID](#method.clearSSID) | Clears the saved SSID |
 | [connect](#method.connect) | Attempts to connect to the specified SSID with the given passphrase |
 | [disconnect](#method.disconnect) | Disconnects from the SSID |
 | [getConnectedSSID](#method.getConnectedSSID) | Returns the connected SSID information |
-| [getCurrentState](#method.getCurrentState) | Returns the current Wifi state |
+| [getCurrentState](#method.getCurrentState) | Returns the current Wifi States |
```

Comment:
This method returns only one state (the current state). It does not return multiple "states". Please revert to "state".